### PR TITLE
Tab ui branch

### PR DESCRIPTION
**Pull request by jynCoder**

### CODE REVIEW

**Comment by v1shanker** [comment]
If this branch still needs to be reviewed please rebase or merge master and then have a podmate review it.

**Comment by jynCoder** [comment]
@googlebot I signed it!

File: src/main/webapp/tab.html [hunk 1]
```diff
@@ -11,7 +11,15 @@
     <div class = "wrap">
         <div class = "navigationBar" id="nav_bar"></div>
         <div class = "bodyMain" id= "body_main"></div>
-            <div class = "leftCol" id = "left_col"></div>
+            <div class = "leftCol" id = "left_col">
+                <div id="mySidebar" class="sidebar">
+                <p class="rep-name">null</p>
+                <a href = "/repList.html" id="repRetButton"><span><i class="material-icons">list</i><span class="repList-text icon-text">Other Representatives</span></a><br>
+                <a href="#" value="Education" id="eduButton" onclick="getTab(this.value)"><span><i class="material-icons">school</i><span class="edu-text icon-text">Education</span></a><br>
```
Comment (jindaln) [inline src/main/webapp/tab.html:18]: These should no longer be hardcoded as tabs are dynamically created - might be better to abstract into a function

File: src/main/webapp/stylesheets/loginStyle.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather:400,400i,700,700i,900,900i');
+
+body{
```
Comment (jindaln) [inline src/main/webapp/stylesheets/loginStyle.css:3]: Check spacing here as well as in .form below

File: src/main/webapp/loginRep.html [hunk 3]
```diff
@@ -3,17 +3,21 @@
...
+    <h5> Weclome to Votify. If you are are representative you are in the right place! Login to your current account
+        to veiw your personal feed of questions from your constituents, or if you are new sign up!</h4>
+    <form class="form" id="loginRep" action="/validate_rep" method='POST'>
         <label for="username">Username: </label>
-        <input type="text" id="username" name="username"><br><br>
+        <input type="text" id="username" name="username" class="inpt" placeholder="ex:George Washington"><br><br>
         <label for="password">Password: </label>
-        <input type="text" id="password" name="password"><br><br>
-        <input onclick="storeRepBooleanAndSubmit();" type="submit">
+        <input type="passwrod" id="password" name="password" class="inpt" placeholder="password"><br><br>
```
Comment (jindaln) [inline src/main/webapp/loginRep.html:17]: type="password"

File: src/main/webapp/stylesheets/feedStyle.css [hunk 4]
```diff
@@ -86,3 +79,111 @@ i {
...
+  /* Support for IE. */
+  font-feature-settings: 'liga';
+}
+
+.rep-name{
+    text-align: center;
+    font-family: Lato;
+    font-size: 25px;
+    color: white;
+}
```
Comment (jindaln) [inline src/main/webapp/stylesheets/feedStyle.css:189]: newline

File: src/main/webapp/stylesheets/feedStyle.css [hunk 5]
```diff
@@ -86,3 +79,111 @@ i {
...
+  color: #f1f1f1;
+}
+
+main .sidebar{
+  position: absolute;
+  padding-top: 60px;
+  right: 25px;
+  font-size: 36px;
+  margin-left: 50px;
+
```
Comment (jindaln) [inline src/main/webapp/stylesheets/feedStyle.css:111]: spacing

File: src/main/webapp/stylesheets/feedStyle.css [hunk 6]
```diff
@@ -86,3 +79,111 @@ i {
...
+  margin-left: 50px;
+  
+}
+
+.icon-text {
+  vertical-align: middle;
+}
+
+.material-icons {
+
```
Comment (jindaln) [inline src/main/webapp/stylesheets/feedStyle.css:119]: spacing

**Comment by jindaln** [comment]
Where are you dynamically loading the tabs - is there a script function for this?

**Comment by jindaln** [comment]
Also please merge master into this - there are some formatting changes that were made to script.js

File: src/main/webapp/loginRep.html [hunk 7]
```diff
@@ -3,17 +3,21 @@
...
     <title>Representative Login Page</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="stylesheets/loginStyle.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <form id="loginRep" action="/validate_rep" method='POST'>
+    <center>
+    <h5> Weclome to Votify. If you are are representative you are in the right place! Login to your current account
+        to veiw your personal feed of questions from your constituents, or if you are new sign up!</h4>
```
Comment (swetha-gangu) [inline src/main/webapp/loginRep.html:12]: check for typos

File: src/main/webapp/loginRep.html [hunk 8]
```diff
@@ -3,17 +3,21 @@
   <head>
     <meta charset="UTF-8">
     <title>Representative Login Page</title>
-    <link rel="stylesheet" href="style.css">
```
Comment (swetha-gangu) [inline src/main/webapp/loginRep.html:6]: We don't seem to have a style.css? I might be missing something but this is giving me error messages when I run in dev

File: src/main/webapp/loginUser.html [hunk 9]
```diff
@@ -3,16 +3,19 @@
   <head>
     <meta charset="UTF-8">
     <title>User Login Page</title>
-    <link rel="stylesheet" href="style.css">
```
Comment (swetha-gangu) [inline src/main/webapp/loginUser.html:6]: Do we have a style.css?

**Pull request closed without merging**